gradle.properties

```diff
@@ -5,7 +5,7 @@ group=com.enonic.app
 projectName=contentstudio.plus
 appName=com.enonic.app.contentstudio.plus
 version=1.5.0-SNAPSHOT
-xpVersion=7.10.0
-libAdminUiVersion=4.3.0
+xpVersion=7.11.0-SNAPSHOT
+libAdminUiVersion=4.4.0-SNAPSHOT
```

Comment:
Do we really need that? If we can avoid dependencies on snapshots, we should not upgrade.

Reply:
Done